.gitlab/package_build/linux.yml

```diff
@@ -83,6 +89,14 @@ datadog-ot-agent-7-x64:
 datadog-ot-agent-7-arm64:
   extends: [.agent_build_common, .agent_build_arm64, .ot_agent_7_build]
 
+# build Agent 7 binaries for x86_64 with FIPS
+datadog-agent-7-x64-fips:
+  extends: [.agent_build_common, .agent_build_x86, .agent_7_build, .agent_fips_build]
+
+# build Agent 7 binaries for arm64 with FIPS
+datadog-agent-7-arm64-fips:
+  extends: [.agent_build_common, .agent_build_arm64, .agent_7_build, .agent_fips_build]
```

Comment:
```suggestion
  extends: [.agent_build_common, .agent_build_arm64, .agent_fips_build]
```

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/8a52d52c113056e5fad1f3d4bfc4968b66b25d2c